### DISCUSSION TITLE
Animation et transition de fichier SVG

### DISCUSSION THREAD

**Discussion by Mica31000** [senshu/Sozi#561] [Q&A] [answered]
Bonjour,

Je souhaite animer des fichiers SVG avec SOZI. L’objectif étant de me rapprocher de l'effet Whiteboard Animation (comme avec le logiciel Videoscribe).

Concrètement j'aimerai à partir d'une forme dessinée avec Inkscape, je souhaiterai y ajouter cet effet (trouvé sur ce site:https://la-cascade.io/ligne-animee-en-svg/): 
`//CSS
.path {
  stroke-dasharray: 1000;
  stroke-dashoffset: 1000;
  animation: dash 5s linear forwards;
}

@keyframes dash {
  to {
    stroke-dashoffset: 0;
  }
}`

Cela est il possible avec Sozi (je me demandai par exemple si l'option feuille de style personnalisée et script pouvait servir à ça)?

Merci de votre aide.

Mica

**Comment by Moini** [comment, marked as answer]
Try https://gitlab.com/Moini/ink_line_animator

**Reply by Mica31000** [reply]
Merci beaucoup! C'est exactement ce que je cherchais!

**Reply by Moini** [reply]
De rien :)
